test/ao/sparsity/test_pruner.py

```diff
@@ -118,13 +121,49 @@ def update_mask(self, layer, **kwargs):
 
 
 class TestBasePruner(TestCase):
-    def test_constructor(self):
-        # Cannot instantiate the base
+    def _is_pruner_prepared(self, model, pruner, device):
+        for g in pruner.module_groups:
+            module = g['module']
+            assert module.weight.device == device
+            # Check mask exists
+            assert hasattr(module, 'mask')
+            # Check parametrization exists and is correct
+            assert parametrize.is_parametrized(module)
+            assert hasattr(module, "parametrizations")
+            # Assume that this is the 1st/only parametrization
+            assert type(module.parametrizations.weight[0]) == PruningParametrization
+        return True
```

Comment:
nit: maybe rename this to something like _check_prepared_pruner` and remove the return, similar for other functions